recipes/jwasm/all/test_package/conanfile.py

```diff
@@ -0,0 +1,10 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch"
+
+    def test(self):
+        if not tools.cross_building(self.settings):
```

Comment:
Perhaps you can copy an assembly file from https://github.com/JWasm/JWasm-regressions?

Reply:
I've added a test on Linux 64 bits, but I can't test it locally right now.

Reply:
done